lib/Gnuplot/Builder.pm

```diff
@@ -294,6 +294,23 @@ Arrange dataset options in valid order.
     $dataset = gfile('data.txt', with => "linespoints", using => "1:3"); ## NG!
     $dataset = gfile('data.txt', using => "1:3", with => "linespoints"); ## OK
 
+=item *
+
+Encode wide characters appropriately.
+
+    use utf8;
+    use Gnuplot::Builder::Process;
+    
+    ## Let Gnuplot::Builder automatically encode input data for gnuplot process
+    $Gnuplot::Builder::Process::ENCODING = 'utf8';
+    
+    $script->set(title => '"日本語のタイトル"');
+    $script->plot('sin(x)');
+
+Otherwise, a warning like C<Wide character in print> would appear. See also: L<Gnuplot::Builder::Process/$ENCODING>.
```

Comment:
What about this alternate wording?
```
When encoding warnings like for example C<Wide character in print>
appear when running your script, it indicates that
L<Gnuplot::Builder::Process> has a different idea of the current
character encoding than your script. In such cases, set the variable
L<Gnuplot::Builder::Process/$ENCODING> to an appropriate value to inform
L<Gnuplot::Builder::Process> about the desired character encoding to use
when sending data to the external Gnuplot process.
```

Reply:
I think this is just too verbose. I think the comment in the example ("## Let Gnuplot::Builder automatically encode input data for gnuplot process") and explanation in `Gnuplot::Builder::Process/$ENCODING` are sufficient.

Reply:
How is it "automatic" when you have to specify `'utf8'`?